Tools/scripts/summarize_specialization_stats.py

```diff
@@ -0,0 +1,41 @@
+"""Print a summary of specialization stats for all files in the
+default stats folders.
+"""
+
+import collections
+import os.path
+
+if os.name == "nt":
+    DEFAULT_DIR = "c:\\temp\\py_stats\\"
+else:
+    DEFAULT_DIR = "/tmp/py_stats/"
+
+
+TOTAL = "deferred", "hit", "miss", "unquickened"
+
+def print_stats(name, stats):
```

Comment:
```suggestion
def print_stats(name, family_stats):
```

(There are two different things called stats in this script).